packages/internal/src/callback.js

```diff
@@ -305,12 +307,14 @@ harden(prepareAttenuator);
 /**
  * Prepare an attenuator whose methodNames are derived from the interfaceGuard.
  *
+ * @template {import('@endo/patterns').InterfaceGuard} G
```

Comment:
this isn't being used as a type parameter. did you mean to use it for the return type?

Reply:
It's being used as the type of the input `interfaceGuard`, and it (via inference) is also used as part of the return type.